build-virtualbox.ps1

```diff
@@ -19,7 +19,12 @@ Test-PluginInstalled "vagrant-winrm"
 Test-PluginInstalled "vagrant-winrm-syncedfolders"
 
 Import-Module PSScriptAnalyzer
-$excludedRules = @('PSUseShouldProcessForStateChangingFunctions', 'PSAvoidUsingPlainTextForPassword', 'PSAvoidUsingUserNameAndPassWordParams', 'PSAvoidUsingConvertToSecureStringWithPlainText')
+$excludedRules = @(
+    'PSUseShouldProcessForStateChangingFunctions', 
+    'PSAvoidUsingPlainTextForPassword', 
```

Comment:
Can we kill this exclude now?

Reply:
Looks like it runs OK without that exclusion. So yeah. Dropping it now.